src/vips-notification-worker/src/test/java/ca/bc/gov/open/pssg/rsbc/vips/notification/worker/OutputNotificationConsumerTest.java

```diff
@@ -1,26 +1,48 @@
 package ca.bc.gov.open.pssg.rsbc.vips.notification.worker;
 
 import ca.bc.gov.open.pssg.rsbc.dps.notification.OutputNotificationMessage;
+import ca.bc.gov.open.pssg.rsbc.vips.notification.worker.sftp.SftpService;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import java.io.ByteArrayInputStream;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class OutputNotificationConsumerTest {
 
+    public static final String CASE_1 = "case1";
     private OutputNotificationConsumer sut;
 
+    @Mock
+    private SftpService sftpServiceMock;
+
+    private ByteArrayInputStream fakeContent() {
```

Comment:
fakeContent?

Reply:
is this a TODO to remove or change later?

Reply:
no, it's for the unit test ;-)